server/style.css

```diff
@@ -466,237 +466,102 @@ font {
-}
-
-.LegalandLicensing {
-  background: #FFF8DC !important;
-  color: #000 !important;
-}
-
-.LibreGraphics {
-  background: #ddd !important;
-  color: #11a311 !important;
-}
-
-.Mentoring {
-  background: #998876 !important;
-  color: #fff !important;
-}
-
-.MySQL {
-  background: #8B47BB !important;
-  color: #fff !important;
-}
-
-.NewtoSCALE {
-  background: #7FFF00 !important;
-  color: #228B22 !important;
+.Observability {
+  background-color: #F7511D;
+  color: #FFF;
 }
-
 .NextGeneration {
-  background: #96f74b !important;
-  color: #000 !important;
-}
-
-.OpenNMS {
-  background: #4D9E00 !important;
-  color: #fff !important; 
+  background-color: #66AEBC;
+  color: #FFF;
 }
-
-.Observability {
-  background: #000 !important;
-  color: #ff0 !important;
+.NixCon {
+  background-color: #A7E5B4;
+  color: #000;
 }
-
-.OpenData {
-  background: #6c6c6c !important;
-  color: #fff !important;
+.OpenSourceAIandAppliedScience {
+  background-color: #BA9F78;
+  color: #FFF;
 }
-
 .OpenGovernment {
-  background: #a6ceff !important;
-  color: #d00 !important;
+  background-color: #6493B9;
+  color: #FFF;
 }
-
-.OpenMedical {
-  background: #00ffd8 !important;
-  color: #000 !important;
+.PostgreSQL {
+  background-color: #F1CC0C;
+  color: #FFF;
 }
-
-.OpenSourceinEnterprises {
-  background: #ffd672 !important;
-  color: #000 !important;
+.SystemsandInfrastructure {
+  background-color: #4B5D6B;
+  color: #FFF;
 }
-
-.OpenSourceSoftwareInEducation {
-  background: #72B20F !important;
-  color: #fff !important;
+.CareerDay{
+  background-color: #8A8C82;
+  color: #FFF;
 }
-
-.openSUSE {
-  background: #669900 !important;
-  color: #000 !important;
+.ReproducibleandImmutableSoftware {
+  background-color: #955E77;
+  color: #FFF;
 }
-
-.PostgreSQL {
-  background: #0AACA0 !important;
-  color: #fff !important;
+.KubernetesCommunityDay {
+  background-color: #6CAF4D;
+  color: #FFF;
 }
-
-.Puppet {
-  background: #D2A3E8 !important;
-  color: #000 !important;
-}
-
-.ScaleU {
-  background: #405E5E !important;
-  color: #fff !important;
+.Workshops {
+  background-color: #3A3A3A;
+  color: #FFF;
 }
-
 .Security {
-  background: #000 !important;
-  color: #00FF00 !important;
+  background-color: #CD5465;
+  color: #FFF;
 }
-
 .Sponsored {
-  background: #007F06 !important;
-  color: #fff !important;
-}
-
-.SystemsandInfrastructure {
-  background: #D2A3E8 !important;
-  color: #000 !important;
-}
-
-.Testing {
-  background: #EA18E5 !important;
-  color: #fff !important;
-}
-
-.SysAdmin {
-  background: #C4C249 !important;
-  color: #000 !important;
+  background-color: #FC8B1C;
+  color: #FFF;
 }
-
 .Ubucon {
-  background: #774022 !important;
-  color: #fff !important;
+  background-color: #774022;
+  color: #FFF;
 }
-
 .UpSCALE {
-  background: #956140 !important;
-  color: #fff !important;
+  background-color: #956140;
+  color: #FFF;
 }
-
-.Youth {
-  background: #098C09 !important;
-  color: #fff !important;
+.FOSS@HOME {
```

Comment:
I'm concerned the use of `@` in the selector name might not be valid. Can you confirm @carlynlee @sarcasticadmin ?

Reply:
thats a good question, let me do some digging

Reply:
From the example that @carlynlee sent over while work through this it does look like `FLOSS@HOME` is not the expected color. Its grey (default) in this screenshot:

![Screenshot 2024-03-04 at 3 31 16 PM](https://github.com/socallinuxexpo/scale-signs/assets/30531572/68d5bc27-8e48-43b5-9bca-ff879b3634ce)

I believe we just need to escape the `@`:

```
.FOSS\@HOME {
```

If you want a quick example in https://jsfiddle.net/ :

html
```
<div class="container">
  <div class="one">
    <p>One</p>
  </div>
  <div class="floss@home">
    <p>floss@home</p> 
  </div>
  <div class="floss@broken">
    <p>floss@broken</p>
  </div>
</div>
```

css:
```
/* For each class, set some colors */
.one {
  background-color: cornflowerblue;
}
.floss\@home {
  color: black;
  background-color: #8ADA20;
}
.floss@broken {
  color: black;
  background-color: #8ADA20;
}
```

Results in:
![ss-202403051709612052](https://github.com/socallinuxexpo/scale-signs/assets/30531572/5e7defff-e944-451b-b686-bf2489646ff6)

Reply:
I believe the selector name is the map key value. My suggestion is to just omit the `@` from the key here https://github.com/socallinuxexpo/scale-signs/blob/e166012f2f5f74fee75d16be98c3edaab9d4163a/server/scroll.php#L50 , making it FOSSHOME` and then match the CSS selector to `FOSSHOME`. Same pattern as how spaces are treated. Probably cleaner than escaping.

Reply:
that makes sense, definitely safer :+1: